dataprofiler/profilers/profiler_options.py

```diff
@@ -887,15 +894,26 @@ def _validate_helper(self, variable_path='TextProfilerOptions'):
             errors.append("{}.stop_words must be None "
                           "or list of strings.".format(variable_path))
 
-        if not isinstance(self.words, BooleanOption):
-            errors.append("{}.words must be a BooleanOption "
-                          "object."
-                          .format(variable_path))
+        if (self.top_k_chars is not None and
+                not isinstance(self.top_k_chars, int)):
+            errors.append("{}.top_k_chars must be None "
+                          "or integer.".format(variable_path))
+
+        if (self.top_k_words is not None and
+                not isinstance(self.top_k_words, int)):
+            errors.append("{}.top_k_words must be None "
+                          "or integer.".format(variable_path))
```

Comment:
Should we also validate that the value is positive? Possibly add a test too for this case?

Reply:
good point, added